platform/ios/src/MGLMapView.mm

```diff
@@ -941,10 +932,54 @@ - (void)updateConstraintsForOrnament:(UIView *)view
     [constraints addObjectsFromArray:updatedConstraints];
 }
 
+- (BOOL)hasAutomaticallyAdjustContentInset {
```

Comment:
nit: I'm having trouble understanding the meaning of the method name.

Can we match the similar view controller method? `automaticallyAdjustsContentInsets`

depending on context?

EDIT: Oh, I see `- (BOOL)automaticallyAdjustsContentInset` - I'm more confused now 😕

Reply:
`hasAutomaticallyAdjustContentInset` is an internal convenience method to avoid code duplication. Since we have two ways to check if we disabled adjusting automatically content insets: using `UIViewController.automaticallyAdjustsScrollViewInsets` (deprecated) and `MGLMapView.automaticallyAdjustContentInset` I am using this method to check these options.